protostar/cheatable_starknet/cheatcodes/expect_events_cairo_cheatcode.py

```diff
@@ -0,0 +1,54 @@
+from typing import Callable
+
+from typing_extensions import NotRequired, TypedDict
+
+from protostar.cheatable_starknet.cheatcodes.cairo_cheatcode import CairoCheatcode
+from protostar.cheatable_starknet.cheaters import CairoCheaters
+from protostar.cheatable_starknet.cheaters.expect_events_controller import (
+    ExpectEventsController,
+    Event,
+)
+from protostar.starknet import CairoData, Address
+from protostar.starknet.selector import Selector
+
+
+class RawExpectedEvent(TypedDict):
+    from_address: int
+    name: str
+    data: NotRequired[CairoData]
+
+
+class ExpectEventsCheatcode(CairoCheatcode):
+    def __init__(self, controller: ExpectEventsController, cheaters: "CairoCheaters"):
+        super().__init__(cheaters)
+        self._controller = controller
+
+    @property
+    def name(self) -> str:
+        return "expect_events"
+
+    def _build(self) -> Callable:
+        def expect_events(
+            *raw_expected_events: RawExpectedEvent,
```

Comment:
this input should be validated

Reply:
It is in the method below.

Reply:
No it's not, 
`raw_expected_event["name"]`
`raw_expected_event["from_address"]`
those will raise `KeyError`

Reply:
Good point